02_production.ipynb

```diff
@@ -29,7 +29,11 @@
    "cell_type": "markdown",
```

Comment:
<p>"make sure if" isn't quite right. Maybe "let's check whether" would be best?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastbook/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>